packages/dds/tree/src/index.ts

```diff
@@ -35,9 +35,25 @@ export {
 export { type Brand } from "./util/index.js";
 
 export type {
+	/**
+	 * Moved to the `@fluidframework/core-interfaces` package.
+	 * @deprecated Moved to the `@fluidframework/core-interfaces` package.
+	 */
 	Listeners,
+	/**
+	 * Moved to the `@fluidframework/core-interfaces` package.
+	 * @deprecated Moved to the `@fluidframework/core-interfaces` package.
+	 */
 	IsListener,
+	/**
+	 * Moved to the `@fluidframework/core-interfaces` package.
+	 * @deprecated Moved to the `@fluidframework/core-interfaces` package.
+	 */
 	Listenable,
+	/**
+	 * Moved to the `@fluidframework/core-interfaces` package.
+	 * @deprecated Moved to the `@fluidframework/core-interfaces` package.
+	 */
 	Off,
```

Comment:
So far, we've advised public customers to only import from these packages: https://learn.microsoft.com/en-us/azure/azure-fluid-relay/concepts/version-compatibility.  This is primarily to simplify usage for customers (as it's difficult/annoying to shop through and install our many packages to find the desired interfaces, and also to help clarify support contracts as detailed on that page.

This is why the other core-interfaces are re-exported as well.  We can always revisit these decisions, but for now this re-export is aligned with our current patterns.  I do think that with this move, they should only be re-exported from core-interfaces and _not_ from tree though.

Reply:
Before 3.0 release, I guess it's ok to remove re-exports from tree. Doing so now would cause type breaks in tree. Instead, the core-interfaces exports from fluid-framework can be removed for now and reintroduce them once the tree re-exports are removed.